setup.py

```diff
@@ -0,0 +1,25 @@
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+setup(
+  name="schemas.zcrypto",
+  version="0.0.1",
+  description="ZSchema definitions for zcrypto's JSON output.",
+  classifiers=[
+    "Programming Language :: Python",
+    "Natural Language :: English"
+  ],
+  author="ZMap Team",
+  author_email="zmap-team@umich.edu",
```

Comment:
this should move to team@zmap.io

Reply:
Will do. I copied this from ztag, so presumably that needs to be updated too?

Reply:
Yeah, we should update it there too.